infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationManager.java

```diff
@@ -189,20 +189,28 @@ public void processStandbyChange(TopologyDescriptor newConfig) {
         Set<String> standbysToRemove = new HashSet<>(currentStandbys);
         standbysToRemove.removeAll(intersection);
 
-        //Remove standbys that are not in the new config
+        // Remove standbys that are not in the new config
         for (String clusterId : standbysToRemove) {
             stopLogReplicationRuntime(clusterId);
             topology.removeStandbyCluster(clusterId);
         }
 
-        //Start the standbys that are in the new config but not in the current config
+        // Start the standbys that are in the new config but not in the current config
         for (String clusterId : newStandbys) {
             if (!runtimeToRemoteCluster.containsKey(clusterId)) {
                 ClusterDescriptor clusterInfo = newConfig.getStandbyClusters().get(clusterId);
                 topology.addStandbyCluster(clusterInfo);
                 startLogReplicationRuntime(clusterInfo);
             }
         }
+
+        // The connection id or other transportation plugin's info could've changed for
```

Comment:
Improper comment style: https://google.github.io/styleguide/javaguide.html#s4.8.6-comments

Reply:
Thanks for pointing it out. Tried to have the same style comments as above. There are many comments in Corfu using improper style, will address them in the future.